src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -3,19 +3,18 @@
 import edu.wpi.first.wpilibj.Preferences;
 
 public class MoPrefs {
-  static final double INTAKE_ROLLER_SETPOINT = 1.0;
-  /**
-   * The maximum accleration of the intake roller, in motor setpoint per scheduler
-   * cycle.
-   */
+  static final double INTAKE_ROLLER_SETPOINT = -0.125;
   static final double INTAKE_ROLLER_ACC_RAMP = 0.1;
   static final int CLIMBER_ENCODER_LIMIT = 10;
-  static final double SHOOTER_HOOD_SETPOINT = 15;
-  static final double SHOOTER_HOOD_POSITION_TOLERANCE = 0.01;
+  static final double SHOOTER_HOOD_SETPOINT = 100;
+  static final double SHOOTER_HOOD_POSITION_TOLERANCE = 2;
   static final double SHOOTER_GATE_SETPOINT = 1;
   static final double SHOOTER_FLYWHEEL_SETPOINT = 1;
+  static final double SHOOTER_PID_SETPOINT = 4500;
   static final double SHOOTER_FLYWHEEL_IDLE = 0.2;
-  static final double STORAGE_SPEED = 0.5;
+  static final double STORAGE_SPEED = 0.75;
+  static final double SHOOTER_FLYWHEEL_TOLERANCE = 100; // RPM
+  static final double AUTO_CHOOSER = 0;
```

Comment:
What's this for?  AutoChooser should be a named pulldown, not numbers in the Prefs.